modules/mesh-layers/src/scenegraph-layer/scenegraph-layer.js

```diff
@@ -260,6 +263,7 @@ export default class ScenegraphLayer extends Layer {
           sizeScale,
           opacity,
           enableOffsetModelMatrix: _composeModelMatrix,
+          sizePixels: [sizeMinPixels, sizeMaxPixels],
```

Comment:
For readability, just send the two uniforms separately as floats.

Reply:
Done